cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -1307,13 +1307,20 @@ public static final class AppMetadataStore {
      * Source Control Management metrics.
      */
     public static final class SourceControlManagement {
-
       public static final String CLONE_REPOSITORY_SIZE_BYTES =
           "source.control.git.repository.clone.size.bytes";
       public static final String CLONE_LATENCY_MS =
           "source.control.git.clone.latency.ms";
       public static final String COMMIT_PUSH_LATENCY_MILLIS =
           "source.control.git.commit.push.duration.ms";
+      public static final String PULL_COUNT =
```

Comment:
nit: PULL_OPERATION_COUNT would be more descriptive and help differentiate from PULL_APP_COUNT